### DISCUSSION TITLE
docker minio  console happened chinese file name Unreadable Code  when  http downloading  file

### DISCUSSION THREAD

**Discussion by awsbamboo115** [minio/minio#15068] [Q&A] [answered]
docker minio  console happened chinese file name Unreadable Code  when  http download  file 
Version  2022-01-04T07:41:07Z

![image](https://user-images.githubusercontent.com/35585886/173013014-01f6ceb5-d063-4945-94ed-3257d0780b0b.png)

**Comment by harshavardhana** [comment, marked as answer]
@awsbamboo115 upgrade your setup

**Comment by awsbamboo115** [comment]
upgrade minio images to minio/minio:RELEASE.2022-06-11T19-55-32Z 
now ,It,s OK

![image](https://user-images.githubusercontent.com/35585886/173504101-018385d7-188c-4059-84c4-f3d8a461a6b4.png)
